test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -683,3 +683,24 @@ Describe "Pwsh exe resources tests" -Tag CI {
         $psversiontable.os | Should -MatchExactly "$($osversion.Major).$($osversion.Minor)"
     }
 }
+
+Describe 'Pwsh startup in directories that contain wild cards' -Tag CI {
+    BeforeAll {
+        $powershell = Join-Path -Path $PsHome -ChildPath "pwsh"
+        $dirnames = "[T]est","[Test","T][est","Test"
+        foreach ( $d in $dirnames ) { new-item -type Directory -path "${TESTDRIVE}/$d" }
+        $testcases = foreach ( $d in $dirnames ) { @{ Dirname = $d } }
+    }
+
+    It "pwsh can startup in a directory named <dirname>" -testcases $testcases {
+        param ( $dirname )
+        try {
+            Push-Location -LiteralPath "${TESTDRIVE}/${dirname}"
+            $result = & $powershell -c '(Get-Item .).Name'
```

Comment:
Should we use cmdlet? Maybe better .Net method to get current directory or -LiteralPath?

Reply:
it really doesn't matter, does it?

Reply:
I agree, maybe use 
```PowerShell
$result = & $powershell -c '([Environment]::CurrentDirectory)'
```

Reply:
also, I want to check only the name,  ason some systems, Environment.CurrentDirectory will include a symlink which is a problem because PS does not, I'm I need to call split-path, I can use this form just as well.

Reply:
My misunderstanding is that we use `Directory.GetCurrentDirectory()` in the code and `Set-Location` and `Get-Item` in the test - do we work with different current directories (first in .Net, second in FileSystem provider)?